README.md

```diff
@@ -38,26 +35,54 @@ Features:
 
 ## Developers
 
-If you want to contribute to this project, you can install this project from source like this :
+If you want to contribute, you can install this project from source like this:
+
 (After cloning the repository)
 
-Enter the virtual environnment :
-```
+### Requirements
+
+This project requires python 3.8 to work!
+
+### 1. Using pipenv
+
+you can use [pipenv](https://pipenv.pypa.io/) (installed through pip) to simplify the process of setting up a virtual environment.
+
+1. Create a virtual environment:
+
+```shell
 pipenv shell
 ```
-if you do not have python 3.8 installed you can try to use another version
-```
+
+if you do not have python 3.8 installed you can use another version like so:
+
+```shell
 pipenv shell --python path/to/python
 ```
 
-Install the dependencies :
-```
+2. Install the dependencies:
+
+```shell
 pipenv install
 ```
 
-Add the current directory to the list of python modules :
-```
+3. Add the current directory to the list of python modules:
+
+```shell
 export PYTHONPATH="`pwd`"
 ```
 
-At this point you should be able to run appimage-builder by running `python appimagebuilder` !
+At this point you should be able to run appimage-builder by running `pipenv run appimagebuilder`!
+
+### 2. Manual Installation
+
+For those who don't want to use `pipenv` there's a `requirements.txt` file that contains all the dependencies.
+
+After installing the required dependencies you can run the project like so:
+
+```shell
+python appimagebuilder/__main__.py
+```
```

Comment:
I did not write a step by step here as i assumed that anyone who wants to go the "manual" route already knows how to use python's venv and the `requirements.txt` file.